python/test/vmafexec_test.py

```diff
@@ -946,66 +946,27 @@ def test_run_vmafexec_runner_akiyo_multiply_no_enhn_gain_model_and_cmd_options(s
             None, fifo_mode=True,
             delete_workdir=True,
             result_store=None,
-            optional_dict={'disable_clip_score': True, 'model_filepath': VmafConfig.model_path("vmaf_v0.6.1neg.json"),
-                           'adm_enhn_gain_limit': 1.2}
+            optional_dict={'no_prediction': True, 'disable_clip_score': True, 'model_filepath': VmafConfig.model_path("vmaf_v0.6.1neg.json"),
+                           'adm_enhn_gain_limit': 1.2, 'vif_enhn_gain_limit': 1.0}
         )
         self.runner.run(parallelize=True)
 
         results = self.runner.results
 
-        try: self.assertAlmostEqual(results[0]['VMAFEXEC_adm2_score'], 1.116595, places=4)  # 1.116691484215469
+        try: self.assertAlmostEqual(results[0]['VMAFEXEC_VMAF_integer_feature_adm2_score_adm_enhn_gain_limit_1.20_score'], 1.116595, places=4)  # 1.116691484215469
         except AssertionError as e: self.verificationErrors.append(str(e))
-        try: self.assertAlmostEqual(results[0]['VMAFEXEC_vif_scale0_score'], 0.983699512450884, places=4)  # 1.0522544319369052
+        try: self.assertAlmostEqual(results[0]['VMAFEXEC_VMAF_integer_feature_vif_scale0_score_vif_enhn_gain_limit_1.00_score'], 0.983699512450884, places=4)  # 1.0522544319369052
         except AssertionError as e: self.verificationErrors.append(str(e))
-        try: self.assertAlmostEqual(results[0]['VMAFEXEC_vif_scale1_score'], 0.9974276726830457, places=4)  # 1.0705609423182443
+        try: self.assertAlmostEqual(results[0]['VMAFEXEC_VMAF_integer_feature_vif_scale1_score_vif_enhn_gain_limit_1.00_score'], 0.9974276726830457, places=4)  # 1.0705609423182443
         except AssertionError as e: self.verificationErrors.append(str(e))
-        try: self.assertAlmostEqual(results[0]['VMAFEXEC_vif_scale2_score'], 0.9984692380091739, places=4)  # 1.0731529493098957
+        try: self.assertAlmostEqual(results[0]['VMAFEXEC_VMAF_integer_feature_vif_scale2_score_vif_enhn_gain_limit_1.00_score'], 0.9984692380091739, places=4)  # 1.0731529493098957
         except AssertionError as e: self.verificationErrors.append(str(e))
-        try: self.assertAlmostEqual(results[0]['VMAFEXEC_vif_scale3_score'], 0.999146211879154, places=4)  # 1.0728060231246508
+        try: self.assertAlmostEqual(results[0]['VMAFEXEC_VMAF_integer_feature_vif_scale3_score_vif_enhn_gain_limit_1.00_score'], 0.999146211879154, places=4)  # 1.0728060231246508
         except AssertionError as e: self.verificationErrors.append(str(e))
 
-        try: self.assertAlmostEqual(results[0]['VMAFEXEC_score'], 122.804272, places=3)  # 132.78849246495625
-        except AssertionError as e: self.verificationErrors.append(str(e))
-
-    def test_run_vmafexec_runner_with_enhn_gain_enabled_disabled(self):
```

Comment:
I'll still be useful to keep this test. Just instead of expecting AssertionError, we expect it to generate legitimate results.